storybook/src/stories/Dropdown.stories.svelte

```diff
@@ -0,0 +1,62 @@
+<script context="module" lang="ts">
+import { Dropdown } from '@podman-desktop/ui-svelte';
+import { type Args, defineMeta, setTemplate, type StoryContext } from '@storybook/addon-svelte-csf';
+
+/**
+ * These are the stories for the `Dropdown` component.
+ */
+const { Story } = defineMeta({
+  component: Dropdown,
+  title: 'Dropdown',
+  tags: ['autodocs'],
+  argTypes: {
+    value: {
+      control: 'text',
+      description: 'Initial value shown in the dropdown',
+      defaultValue: '',
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'Set the dropdown as being disabled',
+      defaultValue: false,
+    },
+    id: {
+      control: 'text',
+      description: 'Id of the dropdown',
+      defaultValue: undefined,
+    },
+    name: {
+      control: 'text',
+      description: 'Name of the dropdown',
+      defaultValue: undefined,
+    },
+  },
+});
```

Comment:
I think it's not telling the callback like
https://github.com/containers/podman-desktop/blob/main/storybook/src/stories/button/Button.stories.svelte#L17-L19

Reply:
Added onChange function to Storybook.